app/templates/sensor-motion.html

```diff
@@ -0,0 +1,9 @@
+<span class="sensor-text-value">
+    <div ng-show="sensor.data.properties.value !== undefined">
+        <span ng-show="sensor.data.properties.value === true" class="balanced">True</span>
```

Comment:
Maybe change these values to human readable, instead of True/False, something like "Movement detected"/"No movement".